docs/content/en/docs-dev/user-guide/adding-an-application.md

```diff
@@ -7,26 +7,52 @@ description: >
 ---
 
 An application is a collect of resources and configurations that are managed together.
-It represents the service which you are going to deploy. With PipeCD, all application's manifests and its application configuration (`.pipe.yaml`) must be committed into a directory of a Git repository. That directory is called as application directory.
+It represents the service which you are going to deploy. With PipeCD, all application's manifests and its application configuration (`app.pipecd.yaml`) must be committed into a directory of a Git repository. That directory is called as application directory.
 
-Before deploying an application, the application must be registered from the web UI and a application configuration file (`.pipe.yaml`) must be committed to the application directory.
-An application must belong to exactly one environment and can be handled by one of the registered `piped`s. Currently, PipeCD is supporting the following kinds of application:
+Before deploying an application, the application must be registered from either the web UI or an application configuration file defined in your Git repository.
+Registering application helps PipeCD know the basic information about that application, where the application configuration is placing, what `piped` should handle it as well as what cloud the application should be deployed to.
+
+An application can be handled by one of the registered `piped`s. Currently, PipeCD is supporting the following kinds of application:
 
 - Kubernetes application
 - Terraform application
 - CloudRun application
 - Lambda application
 - ECS application
 
-## Registering a new application from Web UI
+There are two ways to register an application:
+- From the application configuration in your Git repository (recommended)
+- From the web UI
```

Comment:
I want to say that both of them are "From the web UI", so I guess we need a better way to express them.
Below is just my suggestion.
We can find a better one.

```suggestion
There are two ways to register an application:
- Scanning the unused application configuration files in Git to add (recommended)
- Manually configure application information
```